src/style.css

```diff
@@ -0,0 +1,96 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  height: 403px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 0 14px;
+}
+
+.card__img {
+  margin: 30px 5px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  margin: 13px 0 6px;
+}
+
+.card__product-code {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.stars__item {
+  background: url(/images/star.svg) no-repeat center;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+}
+
+.card__stars .stars__item:nth-child(-n+4) {
```

Comment:
add classes from stars block task, so if I change the class of your element in dev tools I will see how the count of active stars changing.